cellmlmanip/model.py

```diff
@@ -311,30 +327,18 @@ def connect_variables(self, source_name: str, target_name: str):
         return True
 
     def add_rdf(self, rdf: str):
```

Comment:
Are we using typing elsewhere?

Reply:
We were at one point, then stripped it out. Clearly this one was missed :)